src/main/resources/templates/home.html

```diff
@@ -0,0 +1,13 @@
+<html>
+<body>
+<form action="/add-message" method="post">
+    <input type="text" placeholder="message" name="messageText"/>
+    <button type="submit">send</button>
+    <ol>
```

Comment:
D'Angelo the code in your repo is pretty much blank so I don't know if you are having git problems?  If you are let me know so we can solve them.  You don't need to resubmit I saw this project in action in class so I know it works.